src/msg_handlers.ts

```diff
@@ -192,6 +195,27 @@ const makeTestCaseName = (body: any): string => {
   return test_case_name;
 };
 
+export const makeEtaState = (
+  fq_msg: FileQueueMessage,
+): ErrataToolAutomationState => {
+  const { broker_topic, broker_msg_id, body, broker_extra } = fq_msg;
+  var msg_id = broker_msg_id;
+  var version = _.get(body, 'msg_version');
+  var timestamp = Date.parse(_.get(body, 'msg_time'));
```

Comment:
Is it OK that this timestamp is on a different scale than some others we use (milliseconds vs. seconds since the Unix epoch)? `Date.parse` returns milliseconds so I'm just wondering if we should divide it by 1000 or if it's OK since it's not compared anywhere down the line and none of the consumers expect it to be in seconds.

Reply:
Ok, let's make it Unix timestamp

Reply:
Oh, this is hard. I would keet it as it is right now.
https://github.com/hapijs/joi/blob/v9.0.0-3/API.md#datetimestamptype
Until we have actual usage case.

Reply:
Hmm, why can't you just do `Date.parse(_.get(body, 'msg_time')) / 1000`? The message should already be validated by Joi by this point.

Reply:
If we leave this for later, we'll have inconsistent data in the database. Earlier records will use milliseconds and later ones would be with seconds.